modules/nf-core/gatk4/annotateintervals/main.nf

```diff
@@ -32,15 +32,15 @@ process GATK4_ANNOTATEINTERVALS {
     def mappability_track = mappable_regions ? "--mappability-track ${mappable_regions}" : ""
     def segmental_duplication_tracks = segmental_duplication_regions ? "--segmental-duplication-track ${segmental_duplication_regions}" : ""
 
-    def avail_mem = 3
+    def avail_mem = 3072
     if (!task.memory) {
         log.info '[GATK AnnotateIntervals] Available memory not known - defaulting to 3GB. Specify process memory requirements to change this.'
     } else {
-        avail_mem = (task.memory.mega*0.8).intValue()
+        avail_mem = task.memory.mega
```

Comment:
😱 changing it again?

Reply:
Reverted this, since the maxMemPercentage and Xmx don't mix